### DISCUSSION TITLE
能否通过api 或者sdk实现pag导PNG序列帧

### DISCUSSION THREAD

**Discussion by hackry** [Tencent/libpag#2149] [General] [answered]
目前pag导Png序列帧是通过 windows预览工具导出的，能否通过sdk实现pag导png序列帧~

哪怕使用C++ 哪个方法可以实现

**Comment by kevingpqi123** [comment, marked as answer]
可以看下这个实现程序，导出的是一帧帧的 BMP 图片，这里接入一个 png 库，可以直接编码保存成 png 序列
https://github.com/Tencent/libpag/blob/main/linux/src/main.cpp

**Comment by hackry** [comment]
已收到 谢谢！
